example-widget-mui/src/utils.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const isError = (err: unknown) => err instanceof Error;
```

Comment:
May be an implicit rule and I am happy do discuss.
We once said exported functions should define their return types

```suggestion
const isError = (err: unknown): err is Error => err instanceof Error;
```